libs/designsystem/accordion/src/accordion-item.component.spec.ts

```diff
@@ -50,6 +50,14 @@ describe('AccordionItemComponent', () => {
         visibility: 'visible',
       });
     });
+    it('should emit the toggle event when expanded', () => {
+      spyOn(spectator.component.toggle, 'emit');
+      spectator.component.isDisabled = false;
+      spectator.component.isExpanded = false;
+      spectator.component._onToggleExpanded(new KeyboardEvent('keydown'));
```

Comment:
Consider using `spectator.click` method instead to actually test that clicking the item triggers the event (calling `_onToggleExpanded` explicitly doesn't test that the method is wired up to the markup):
```suggestion
      spectator.click('.header');
```